src/blocks/recommendations/components/parsely-recommendations-list.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * External dependencies
+ */
+import classNames from 'classnames';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import ParselyRecommendationsListItem from './parsely-recommendations-list-item';
+
+const ParselyRecommendationsList = ( { imagestyle, recommendations, showimages } ) => (
+	<ul
+		className={ classNames(
+			'parsely-recommendations-ul'
```

Comment:
```suggestion
			'parsely-recommendations-list'
```

The person writing the CSS shouldn't really care if this is a `ul` or `ol`. Chances are, the returned articles **_are_** ordered in some way, so maybe `ol` is correct for the element anyway?

Reply:
I renamed the class with the `list` suffix, but haven't updated the`ul` to an `ol`. There is some logic from the Parse.ly API that obviously sorts the items in the specific order, but I'm unsure I would go for this with a `ol`. Any additional opinions/feedback welcome.